From 6fd301895a39fc32f302ea3fced74fc03e5c75c9 Mon Sep 17 00:00:00 2001
From: Vincent Untz <vuntz@gnome.org>
Date: Mon, 29 Nov 2010 21:47:00 +0000
Subject: capplet: Also give the dialog a reasonable width

We don't want to depend on the natural size, which depends on the length
of strings for this.

https://bugzilla.gnome.org/show_bug.cgi?id=635891
---
diff --git a/capplet/gsm-properties-dialog.c b/capplet/gsm-properties-dialog.c
index 8b51169..487250e 100644
--- a/capplet/gsm-properties-dialog.c
+++ b/capplet/gsm-properties-dialog.c
@@ -784,7 +784,7 @@ gsm_properties_dialog_init (GsmPropertiesDialog *dialog)
                                                      "main-notebook"));
         gtk_box_pack_start (GTK_BOX (content_area), widget, TRUE, TRUE, 0);
 
-        gtk_window_set_default_size (GTK_WINDOW (dialog), -1, 450);
+        gtk_window_set_default_size (GTK_WINDOW (dialog), 600, 450);
         gtk_window_set_resizable (GTK_WINDOW (dialog), TRUE);
         gtk_container_set_border_width (GTK_CONTAINER (dialog), 6);
         gtk_box_set_spacing (GTK_BOX (content_area), 2);
--
cgit v0.9.0.2
